compiler/resolution/functionResolution.cpp

```diff
@@ -5889,6 +5878,7 @@ static FnSymbol* resolveForwardedCall(CallInfo& info, check_state_t checkState)
       bestBlock->flattenAndRemove();
     }
   }
+  forwardCallCycleSet.clear();
```

Comment:
So is the gist that if we reach this point we don't think there are any cycles left to detect (while noting the "not perfect" disclaimer above)?